holoviews/plotting/bokeh/element.py

```diff
@@ -104,6 +103,12 @@ class ElementPlot(BokehPlot, GenericElementPlot):
           * timeout   - Timeout (in ms) for checking whether interactive
                         tool events are still occurring.""")
 
+    sizing_mode = param.ObjectSelector(default='fixed',
+                                       objects=[
+            'fixed', 'scale_width', 'scale_height',
+            'scale_both', 'stretch_both'
+        ], doc="""Defines how the plot scales when resized.""")
+
```

Comment:
>That is probably why what Jim suggests makes the most sense - just say you aren't ready to receive comments just yet at the top.

Agreed, this PR has been around a while and I forgot to update the description. That said I'm still pretty firm on the meanings of ``WIP`` and ``ready``, but without putting that information somewhere I guess no one will know what those conventions are.